### DISCUSSION TITLE
bug

### DISCUSSION THREAD

**Discussion by 321scooby123** [AK-Homberger/Alexa-Alarm-System-ESP32CAM#3] [General]
Hi, I'm Brazilian, I managed to "translate" a lot and run the code, everything worked, but when I enter the ip where I would have the camera view, it only appears "Not Found (espalexa)" on a black background.
I tried to solve it myself and I couldn't, could you help me?

**Comment by AK-Homberger** [comment]
You have to use "IP-address:90" to start the interface. Port 80 is used for the Espalexa service.

Or alternatively "alarmsystem.local:90" if mDNS (Bonjour) is supported for your browser.

**Reply by 321scooby123** [reply]
they are configured for port 90, but even so, it gives the same error, can I send the code here for you to take a look?

**Reply by 321scooby123** [reply]
I managed to solve it, thank you very much, taking the opportunity to take another doubt, to be able to see the interface, you need to be connected to the same wifi as the esp32, isn't there any way to see the image while away from home?

**Comment by AK-Homberger** [comment]
You can use a VPN connection to see the image: https://github.com/AK-Homberger/Alexa-Alarm-System-ESP32CAM#vpn-alternatives
Depending on your router model and IP version this might be easy or very difficult.
